action.yml

```diff
@@ -0,0 +1,16 @@
+name: 'setup-testspace'
+description: 'Action to install and configuration the Testspace client'
+author: 'S2 Technologies, Inc.'
+inputs:
+  subdomain:  
+    description: 'Testspace subdomain'
+    required: true
+  token:
```

Comment:
Is this how we should do this? If they have TESTSPACE_TOKEN set as an environment variable for the CI don't we automatically pick it up? Do we want to encourage users to check in tokens? I notice that we do this in the help as well, but if the TESTSPACE_TOKEN is set in environment the client automatically uses, correct @ivailop?

Reply:
then Actions pattern is the following:
<pre>
  token:  ${{ secrets.TESTSPACE_TOKEN }}
</pre>

Refer to the README.md usage example.

Reply:
OK, I see this is not what runs the action, but some other config file. Part of the confusion is how you did it here. If I understand correctly then updating yaml file in the step https://github.com/s2technologies/help.testspace.com/blob/master/docs/publish/getting-started-setup.md#private-repo is unnecessary and not sure if even supported by action.

Reply:
for private repos the token is required. The action code sets the "passed" token to `TESTSPACE_TOKEN`.  The name used by the secret does not matter.